doc/userguide/performance/analysis.rst

```diff
@@ -0,0 +1,71 @@
+Performance Analysis
+=====================
+
+There are many possibilities that could be the reason for performance issues.
+In this section we will guide you through some options.
+
+General
+-------
+
+First of all you should check all the log files with a focus on stats.log and
+suricata.log if any obvious issues are seen. There are several tools that can
+help to find a root cause.
+
+A first step is to run a tool like **htop** to get an overview of the system
```

Comment:
Its a bit confusing that we have 'a first step' here, while the previous block said also that we need to do some other things 'First of all'.

Reply:
good point, will rewrite it